src/main/java/edu/umich/ctools/sectionsUtilityTool/SISPollingThread.java

```diff
@@ -0,0 +1,221 @@
+package edu.umich.ctools.sectionsUtilityTool;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.HttpGet;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.activation.DataHandler;
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import javax.mail.util.ByteArrayDataSource;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Created by pushyami on 4/7/17.
+ */
+public class SISPollingThread implements Runnable {
+	private static Log M_log = LogFactory.getLog(SISPollingThread.class);
+
+	volatile boolean flag = true;
+	Properties appExtPropertiesFile = SectionsUtilityToolServlet.appExtPropertiesFile;
+	private int pollingAttempts =Integer.valueOf(appExtPropertiesFile.getProperty(Utils.SIS_POLLING_ATTEMPTS));
+	private int pollingFrequency =Integer.valueOf(appExtPropertiesFile.getProperty(Utils.SIS_POLLING_FREQUENCY));
+
+
+	@Override
+	public void run() {
+		while (flag) {
+			M_log.debug("***************************** Starting polling Thread");
+			List<SISDataHolderForEmail> pollingId = SectionsUtilityToolServlet.canvasPollingIds;
```

Comment:
The names should both be plural or both singular (just like the underlying values).